docs/source/docker/integration/docker_trigger_from_api.rst

```diff
@@ -194,16 +194,16 @@ To see the progress of your docker run, go to the Lightly Platform and head to
 .. image:: ../getting_started/images/docker_runs_overview.png
 
 
-Use your subsampled dataset
+Use your selected dataset
 ---------------------------
 
-Once the docker run has finished, you can see your subsampled dataset in the 
+Once the docker run has finished, you can see your selected dataset in the
```

Comment:
see your selected dataset in -> see your dataset with the selected images in

Reply:
see https://github.com/lightly-ai/lightly/pull/735#discussion_r845343726